apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.ts

```diff
@@ -475,7 +484,7 @@ export class AdvanceReceiptAdjustmentComponent implements OnInit {
             });
             // this.adjustPayment.balanceDue = Number(this.adjustPayment.grandTotal.) - Number(totalAmount);
             this.adjustPayment.totalAdjustedAmount = Number(totalAmount);
-            if (this.adjustPayment.grandTotal - this.adjustPayment.totalAdjustedAmount < 0) {
+            if (this.adjustPayment.grandTotal - this.adjustPayment.totalAdjustedAmount - this.depositAmount + this.adjustPayment.tdsTotal + this.adjustPayment.tcsTotal < 0) {
```

Comment:
Consider moving this expression into a typescript getter that will help to reduce the expression size and make it more readable